packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md

```diff
@@ -5,6 +5,8 @@ title: Deprecated JavaScript methods in the Sign-In Widget
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
+This guide covers the Javascript methods that are deprecated from the Sign-In Widget and describes how to use the `showSignIn()` method instead.
```

Comment:
```suggestion
This guide covers the Javascript method that is deprecated from the Sign-In Widget and describes how to use the `showSignIn()` method instead.
```
We left the title plural in case other deprecated methods come up. But the guide, as it is right now, only covers one deprecated method.